pkgs/development/libraries/indicator-application/gtk2.nix

```diff
@@ -26,8 +26,10 @@ stdenv.mkDerivation rec {
       --replace 'DBUSSERVICEDIR=`$PKG_CONFIG --variable=session_bus_services_dir dbus-1`' \
                 "DBUSSERVICEDIR=$out/share/dbus-1/services"
     autoconf
-    substituteInPlace {configure,ltmain.sh,m4/libtool.m4} \
+    for f in {configure,ltmain.sh,m4/libtool.m4}; do 
+      substituteInPlace $f \
       --replace /usr/bin/file ${file}/bin/file
```

Comment:
I think it might be better to read when the `--replace` expression is indented as well:

``` shell
for f in ...;do
     substitueInPlace $f \
          --replace ...
done
```